Source/WebCore/page/Quirks.cpp

```diff
@@ -2293,6 +2298,9 @@ static void handleGoogleQuirks(QuirksData& quirksData, const URL& quirksURL, con
     // docs.google.com https://bugs.webkit.org/show_bug.cgi?id=161984
     quirksData.isTouchBarUpdateSuppressedForHiddenContentEditableQuirk = quirksData.isGoogleDocs;
 #endif
+#if ENABLE(MEDIA_STREAM)
+    quirksData.shouldEnableEnumerateDeviceQuirk = true;
```

Comment:
Do we need to enable this for all Google properties?

Reply:
I'll change to Google Meet for now